telegram-bot/meetup.go

```diff
@@ -90,6 +120,23 @@ func nextmeetup(ID int64, client mongo.Client) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	nxtmeetupdata := "Details of next OSDC Meetup :" + "\n" + "Title -" + "\t" + result.Name + "\n" + "Date -" + "\t" + result.Date + "\n" + "Venue -" + result.Venue
-	bot.Send(tbot.NewMessage(ID, nxtmeetupdata))
+	timeString := result.Date.Local().Format("Mon Jan _2 15:04:05 2006")
```

Comment:
Please add comments here about what is the purpose of `timeString`.

Reply:
Done. Thanks for bringing that to my attention. :relaxed: 
```suggestion
      //we have to use a sample string to specify the time format to time.Format() 
	timeString := result.Date.Local().Format("Mon Jan _2 15:04:05 2006")
```